docs/source/troubleshooting/tracing.rst

```diff
@@ -0,0 +1,106 @@
+==================================================
+Tracing on NI Linux Real-Time
+==================================================
+
+.. contents:: Table of Contents
+   :depth: 2
+   :local:
+
+Introduction
+============
+
+Often times when debugging latency issues, race conditions or hangs on real-time embedded targets it is useful to know what the kernel/OS was doing just prior to the error condition. Acquiring and analyzing a kernel event or function trace can provide insights into latency spikes, race conditions, lock contention, priority inversions, missed loop iterations and so on.
+
+On NI Linux Real-Time (NILRT) there are several higher level tools that can be used for this like the "LabVIEW Execution Trace Toolkit". Documentation for using those tools is available elsewhere. This document only covers using the lower level APIs for interacting with the kernel ftrace system.
```

Comment:
Do we recommend using the execution trace tookit? If we do, then we should link to its documentation. If not, then we should drop mentioning it here and keep focus on what this doc *is* about.

Reply:
RTETT has been broken for a while but with some manual fixes it can be made to work. But those VIs are going to be replaced in next release. Removed the whole section.